packages/ra-ui-materialui/src/layout/AppBar.js

```diff
@@ -110,7 +110,7 @@ const AppBar = props => {
                                 ? 'ra.action.close_menu'
                                 : 'ra.action.open_menu',
                             {
-                                _: 'Open/Close menu',
```

Comment:
No, the idea of the `_` parameter is to display a string if the actual translation is missing. This is done as a backwards compatibility measure since the text wasn't translated before.

Reply:
Make sense. As you wish.